projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/usage-panel/index.tsx

```diff
@@ -97,8 +100,28 @@ const useUpgradeButtonText = ( planType: PlanType, nextTierRequestLimit: number
 	);
 };
 
+/**
+ * Helper to get the Contact Us URL
+ * @returns {object} an object with the Contact Us URL, the autosaveAndRedirect function and a boolean indicating if we are redirecting
+ */
+const useAIContactUs = (): {
```

Comment:
Mind the lower cap `i` on `Ai`. We're seeing refactors just to get this caps style right.

Reply:
I just removed the AI part to make it more simple and descriptive, `useContactUsLink`.